petermwash/JavaDevsNairobi
# #156008359 Share functionality

#### What does this PR do?
- Add share functionality.

#### Description of Task to be completed?
- Attach a listener to the shareButton.
- Start sharing intent on button click.
- The message of the share should be `Check out this awesome developer @<github username>, <github profile url>`.

#### How should this be manually tested?
- Clone this [repo](https://github.com/petermwash/JavaDevsNairobi) and run the project on an emulator or device.
- Navigate to the user Details screen then click on the share button, you will be able to choose which share service to use as shown in the screenshot below.
- When you choose the sevice to  use, by deafult, it should show the share message as shown in the secreenshot below.

#### Any background context you want to provide?
- No.

#### What are the relevant pivotal tracker stories?
[#156008359](https://www.pivotaltracker.com/story/show/156008359)

#### Screenshots (if appropriate)
Choose the share service to use.
<img width="360" alt="screenshot 2018-11-02 at 10 58 17" src="https://user-images.githubusercontent.com/27728464/47900733-0166bd00-de8f-11e8-81ae-0773bb4ac578.png">

Message for sharing.
<img width="371" alt="screenshot 2018-11-02 at 10 58 52" src="https://user-images.githubusercontent.com/27728464/47900788-3a069680-de8f-11e8-896d-b36044054553.png">

#### Questions:
No.
